shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
+            'actionUpdateQuantity',
```

Comment:
Interesting, these two hooks should NOT have different content in their param `$param`. In this case, your version is acceptable.